src/components/Table/GridTable.tsx

```diff
@@ -599,17 +643,28 @@ const VirtualRoot = memoizeOne<
   ) => Components["List"]
 >((gs, columns, id, firstLastColumnWidth, xss) => {
   return React.forwardRef(({ style, children }, ref) => {
+    // This VirtualRoot list represent the header when no styles are given. The
+    // table list generally has styles to scroll the page for windowing.
+    const isList = Object.keys(style || {}).length !== 0;
```

Comment:
Sadly this is the only way to determine if the List being rendered is the "sticky" list or the non header list.

Reply:
Good idea!